resourceIndexer/inventory.js

```diff
@@ -40,14 +42,14 @@ const fetchInventoryRemote = async ({bucket, prefix, name, save}) => {
       resolve(orderedKeys[0])
     });
   });
-  logger.info(`inventory for ${name} - manifest key: ${manifestKey}`)
+  console.log(`inventory for ${name} - manifest key: ${manifestKey}`)
```

Comment:
From reviewing the `debug` docs (new library to me, looks useful!), it's possible to rebind the output of a debugger to STDOUT instead of STDERR.

I wonder if a pattern like this would be useful? Being able to control debug and info output with the same env var feels like a win to me. 

```suggestion
  const info = debugFactory("resource-indexer:inventory:info");
  info.log = console.log.bind(console);
  // this goes to STDOUT
  info(`inventory for ${name} - manifest key: ${manifestKey}`)
```

Reply:
I'm skeptical of that approach. It's essentially a reinvention of log levels but with log levels that aren't a first-class concept but instead smuggled into the logger name (`…:info`). Controlling an `info` log level via an env var named `DEBUG` is also a bit fishy to me.

If we want this level of logging control, we should use a proper logging library (as in #511).

Reply:
Fair enough. `console.log()` gets zero granular control (other than a `> /dev/null` or whatever...) and I was trying to think of ways to have a more uniform interface. (_Maybe_ also a visceral reaction to `console.log` 😁 )